lenovo/thinkpad/x230/default.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib; {
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop/hdd
+  ];
+
+  boot = {
+    kernelModules = [
+      "kvm-intel"
+      "acpi_call"
+      "tpm-rng"
+    ];
+    extraModulePackages = with config.boot.kernelPackages; [
+      acpi_call
```

Comment:
Have you seen `thinkpad_acpi` module? It might be better for ThinkPads.

http://www.thinkwiki.org/wiki/Thinkpad-acpi
https://github.com/torvalds/linux/blob/0192f17529fa3f8d78ca0181a2b2aaa7cbb0784d/drivers/platform/x86/Kconfig#L399

Reply:
i had some issues with thinkpad_acpi, especially with tlp  but i am willing to give it another whirl.

Reply:
If you have issues, then `acpi_call` is better, at least for now. I'll test both `acpi_call` and `thinkpad_acpi` later (probably right after New Year) and I'll ping you with my results.

Reply:
update:
`tlp stat` without `acpi_call` loaded:
```
...
+++ ThinkPad Battery Features
tp-smapi   = inactive (unsupported hardware)
tpacpi-bat = inactive (kernel module 'acpi_call' not installed)

+++ Battery Status
/sys/class/power_supply/BAT0/manufacturer                   = LGC
/sys/class/power_supply/BAT0/model_name                     = 45N1029
/sys/class/power_supply/BAT0/cycle_count                    = (not supported)
/sys/class/power_supply/BAT0/energy_full_design             =  93240 h
/sys/class/power_supply/BAT0/energy_full                    =  69500 h
/sys/class/power_supply/BAT0/energy_now                     =  62310 h
/sys/class/power_supply/BAT0/power_now                      =  13781 [mW]
/sys/class/power_supply/BAT0/status                         = Discharging

Charge                                                      =   89.7 [%]
Capacity                                                    =   74.5 [%]

+++ Suggestions
* Install acpi-call kernel module for ThinkPad battery features
```

`thinkpad_acpi` gets loaded automatically by other modules without explicitly adding it to the kernelModules.